lib/loader.js

```diff
@@ -16,10 +16,14 @@ module.exports = function(content, map) {
 	var root = query.root;
 	var moduleMode = query.modules || query.module;
 	var camelCaseKeys = query.camelCase || query.camelcase;
+	var sourceMap = query.sourceMap || false;
 	var resolve = createResolver(query.alias);
 
-	if(map !== null && typeof map !== "string") {
+	if(sourceMap && map !== null && typeof map !== "string") {
```

Comment:
I think, the condition should look like:

```javascript
if(sourceMap) {
    if(map && typeof map !== "string") {
        map = JSON.stringify(map);
    }
} else {
    map = null;
}
```

Of course, you can avoid the nesting, but writing it like this makes it pretty clear ... at least to me 😁